### PR TITLE
PR_TEMPLATE_ADDED

### PR DESCRIPTION
**Pull request by vish-han**
the PR TEMPLATE has been added in the .github directory . @anubha-v-ardhan please review the code and let me know if you want some changes.

### CODE REVIEW

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]
Folks will be making pull requests for tweets (temporary plan untill we make issue based workflow), so I think, we need the template related to that.

**Comment by vish-han** [comment]
Can u suggest what changes I should make?

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] This change requires a documentation update
+
```
Comment (anubha-v-ardhan) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:16]: We should add something like "New Tweet (Adding a new `*.tweet` file to tweet from KubeSimplify's Twitter" as currently we only have PR based way of tweeting

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md (hunk 1 above)
Comment (vish-han) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:16]: Ok working on it.👍

Comment (saiyam1814) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:16]: @vish-han you are done with your changes?

Comment (vish-han) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:16]: Some changes are required bhia, working on it .

**Comment by saiyam1814** [comment]
@anubha-v-ardhan we don't need a PR template as PR will be created from the issue itself. right ?

**Comment by anubha-v-ardhan** [comment]
> @anubha-v-ardhan we don't need a PR template as PR will be created from the issue itself. right ?

Yeah, This was a temporary plan when we only had "PR to tweet" support. I think we can close this now

**Pull request closed without merging**